src/main/java/com/isa/morswiny/Main.java

```diff
@@ -1,12 +1,20 @@
 package com.isa.morswiny;
 
 import com.isa.morswiny.eventsDao.EventCRUDRepository;
+import com.isa.morswiny.users.User;
+import com.isa.morswiny.usersDao.UserDataManagement;
+import com.isa.morswiny.usersDao.UserRepository;
 
 public class Main {
     public static void main(String[] args) {
 
-        EventCRUDRepository er = new EventCRUDRepository();
-        System.out.println(er.getEventByID(71944));
+        //EventCRUDRepository er = new EventCRUDRepository();
+        //System.out.println(er.getEventByID(71944));
 
+        UserDataManagement udm = new UserDataManagement();
+        for (User user : udm.getUsersList()) {
+            System.out.println(user.toString());
+
+        }
```

Comment:
ten main to rozumiem do testow i tak naprawde ta logika tutaj niepotrzebna? Jesli tak, to pamietaj prosze  by usunac w przyszlosci